getting-started/add-test-and-security.hbs.md

```diff
@@ -376,7 +376,7 @@ Where `DIGEST` is the component version or image digest printed in the `KUBECTL
 For additional information and examples, see [Tanzu Insight plug-in overview](../cli-plugins/insight/cli-overview.md).
 <br>
 
-Congratulations! You have successfully added testing and security scanning to your application on the Tanzu Application Platform.
+Congratulations! You have successfully added testing and security scanning to your application on the Tanzu Application Platform. See component documentation for [Out of the Box Supply Chain with Testing and Scanning](../scc/ootb-supply-chain-testing-scanning.hbs.md#a-idcve-triage-workflowa-cve-triage-workflow) on the CVE triage workflow.
```

Comment:
Is this anchor link right? The id in the target file doesn't have `a-id`, just: `cve-triage-workflow`.